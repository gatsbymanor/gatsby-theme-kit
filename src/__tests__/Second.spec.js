import React from 'react';
import renderer from 'react-test-renderer';
import Second from '../components/Second';

describe('<Second />', () => {
  const component = renderer.create(<Second />).toJSON();

  it('should render correctly', () => {
    expect(component).toMatchSnapshot();
  });
});