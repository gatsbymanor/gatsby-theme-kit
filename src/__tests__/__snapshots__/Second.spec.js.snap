// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Second /> should render correctly 1`] = `
<section
  className="main special"
  id="second"
>
  <header
    className="major"
  >
    <h2>
      Ipsum consequat
    </h2>
    <p>
      Donec imperdiet consequat consequat. Suspendisse feugiat congue
      <br />
      posuere. Nulla massa urna, fermentum eget quam aliquet.
    </p>
  </header>
  <ul
    className="statistics"
  >
    <li
      className="style1"
    >
      <span
        className="icon fa-code-fork"
      />
      <strong>
        5,120
      </strong>
       Etiam
    </li>
    <li
      className="style2"
    >
      <span
        className="icon fa-folder-open-o"
      />
      <strong>
        8,192
      </strong>
       Magna
    </li>
    <li
      className="style3"
    >
      <span
        className="icon fa-signal"
      />
      <strong>
        2,048
      </strong>
       Tempus
    </li>
    <li
      className="style4"
    >
      <span
        className="icon fa-laptop"
      />
      <strong>
        4,096
      </strong>
       Aliquam
    </li>
    <li
      className="style5"
    >
      <span
        className="icon fa-diamond"
      />
      <strong>
        1,024
      </strong>
       Nullam
    </li>
  </ul>
  <p
    className="content"
  >
    Nam elementum nisl et mi a commodo porttitor. Morbi sit amet nisl eu arcu faucibus hendrerit vel a risus. Nam a orci mi, elementum ac arcu sit amet, fermentum pellentesque et purus. Integer maximus varius lorem, sed convallis diam accumsan sed. Etiam porttitor placerat sapien, sed eleifend a enim pulvinar faucibus semper quis ut arcu. Ut non nisl a mollis est efficitur vestibulum. Integer eget purus nec nulla mattis et accumsan ut magna libero. Morbi auctor iaculis porttitor. Sed ut magna ac risus et hendrerit scelerisque. Praesent eleifend lacus in lectus aliquam porta. Cras eu ornare dui curabitur lacinia.
  </p>
  <footer
    className="major"
  >
    <ul
      className="actions"
    >
      <li>
        <a
          className="button"
          href="generic.html"
        >
          Learn More
        </a>
      </li>
    </ul>
  </footer>
</section>
`;
